src/index.ts

```diff
@@ -38,7 +39,13 @@ export async function releaseItAction({
 
 	await $$`git config user.email ${gitUserEmail}`;
 	await $$`git config user.name ${gitUserName}`;
-	await $$`npm config set //registry.npmjs.org/:_authToken ${npmToken}`;
+	if (npmToken) {
+		await $$`npm config set //registry.npmjs.org/:_authToken ${npmToken}`;
+	} else {
+		core.info(
+			"No npm token provided!  This is required unless you're using Trusted Publishing.",
```

Comment:
[Style] Quick nit: an extra space, and if it's just `info` I don't think all users will need the "!" level of emphasis.

```suggestion
			"No npm token provided. This is required unless you're using Trusted Publishing.",
```

Reply:
Heh, I'm showing my age.  When I took typing class in middle school they taught us to use two spaces between sentences.  And that's still my muscle memory 👴